components/collections/benches/codepointtrie.rs

```diff
@@ -62,6 +62,17 @@ fn overview_bench(c: &mut Criterion) {
         });
     });
 
+    c.bench_function("cpt/get_range", |b| {
```

Comment:
Benches other than the overview bench should be gated: 
```suggestion
    #[cfg(feature = "bench")]
    c.bench_function("cpt/get_range", |b| {
```